serverless/metering-serverless.adoc

```diff
@@ -0,0 +1,62 @@
+[id="metering-serverless"]
+= Using metering with {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: metering-serverless
+
+As an {product-title} cluster administrator, you can use Metering to analyze what is happening in your {ServerlessProductName} cluster.
+
+.Additional resources
+For more information about Metering on {product-title}, see xref:../metering/metering-about-metering.adoc[About Metering].
+
+[NOTE]
+====
+Some of the following instructions use the `openshift-metering` project and namespace, however for use with {ServerlessProductName}, you should use `metering` as the project and namespace.
+====
+
+[id="installing-metering-serverless_{context}"]
+== Installing Metering
+For information about installing Metering on {product-title}, see xref:../metering/metering-installing-metering.adoc[Installing Metering].
+
+[id="datasources-metering-serverless_{context}"]
+== Applying Datasources for Knative Serving Metering
+The following `ReportDataSources` are available for use with Metering and the {ServerlessProductName} Knative Serving component:
+
+`knative-service-cpu-usage`:: Accumulated CPU seconds used per Knative service over the report time period.
+`knative-service-memory-usage`:: Average Memory consumption per Knative service over the report time period.
+
+These `ReportDataSources` are found in the `datasources` directory.
+You can apply them by using the following command:
+
+```
+$ oc apply -f datasources/
+```
+
+[id="queries-metering-serverless_{context}"]
+== Applying Queries for Knative Serving Metering
+The predefined `ReportQuery` resources are stored in the `queries` directory and can be applied using the command:
+
+```
+$ oc apply -f queries/
+```
+
+[id="reports-metering-serverless_{context}"]
+== Running a Metering report for Knative Serving
+You can run Metering reports against Knative Serving by creating `Report` resources.
+You can find example reports located in the `reports` directory.
+Before you run a report, you must modify the input parameter within the `Report` resource to specify the start and end dates of the reporting period.
+
+Once you have provided the input parameters, you can run the report using the command:
+```
+$ oc apply -f reports/<report-name>.yml
+```
+You can then check the report as shown in the following example:
+```
+$ kubectl get report
+
+NAME                        QUERY                       SCHEDULE   RUNNING    FAILED   LAST REPORT TIME       AGE
+knative-service-cpu-usage   knative-service-cpu-usage              Finished            2019-06-30T23:59:59Z   10h
+```
+
+// Viewing metering reports
+include::modules/metering-viewing-report-results.adoc[leveloffset=+1]
```

Comment:
The preview has one occurence of `{product-tile} ` which doesn't resolve (you can see it when looking at the preview)

Reply:
The last example output in the included file will look differently for Knative. Please look at the tutorials from @rhuss and see they're a little different.

Reply:
I will just rewrite the content in this case then, if our version is different we can't really re-use the content.

Reply:
Should be fixed now @mgencur - I removed this section and replaced it